04.ls/ls.rb

```diff
@@ -1,43 +1,135 @@
 # frozen_string_literal: true
 
 require 'optparse'
+require 'etc'
 
 COLUMN = 3
-SPACE = 2
+FORMAT_TIME = '%b %d %H:%M'
+BLOCK_SIZA = 1024
```

Comment:
```suggestion
BLOCK_SIZE = 1024
```

かな？

スペルチェッカーを入れておくといいかもしれません。

[Code Spell Checker でスペルミスをチェックする \[VSCode\] \- もとひろ blog](https://motohiro-mm.hatenablog.com/entry/2023/10/11/131354)

Reply:
ありがとうございます🙇‍♂️
スペルチェッカー導入します！